_dashboards/search-telemetry.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Search telemetry
+nav_order: 30
+---
+
+
+# About search telemetry
+
+You can use search telemetry to analyze performance for success or failed search requests in OpenSearch Dashboards. OpenSearch stores telemetry data in the `.kibana_1` index.
+
+Because there are thousands of concurrent search request from OpenSearch Dashboards, the large traffic causes significant load in an OpenSearch cluster.
+
+By default it is disabled, so you need to do some configuration changes to enable it.
+
+OpenSearch clusters perform better with search telemetry turned off.
+{: .tip }
+
+To learn more about using telemetry data with OpenSearch Dashboards, see [Trace Analytics OpenSearch Dashboards plugin](docs/2.0/observability-plugin/trace/ta-dashboards/).
+
+## Enable search telemetry
+
+Search usage telemetry is disabled by default. To enable it, you need to set the `data.search.usageTelemetry.enabled` setting to `true` in the `opensearch_dashboard.yml` file.
+
+You can find the OpenSearch Dashboards YAML file in the opensearch-project GitHub directory: `OpenSearch-Dashboards/config/opensearch_dashboards.yml`
+{: .note }
+
+Alternatively, you can modify the Data plugin config file to opt out of search telemetry.
+
+### To opt-in or opt-out of search telemetry data
+
+You can opt-in or opt-out of using search telemetry in your cluster by changing the configuration values in both the OpenSearch Dashboards YAML and Data plugin configuration files.
+
+The following table shows the combination of values for the OpenSearch Dashboards YAML file setting `data.search.usageTelemetry.enabled` and the Data plugin configuration file OpenSearch-Dashboards/src/plugins/data/config.ts setting `usageTelemetry` values that will result in search telemetry opt-in or opt-out.
```

Comment:
Need a little more info about what the Data plugin is. It's not currently documented https://opensearch.org/docs/latest/dashboards/install/plugins/

If it was added without our knowledge, we should document it first before adding this setting.

Reply:
thanks, yes it was very confusing. the issue said they had to update both YAML and "data plugin" config file. 
but after a few more questions from requester, it is clarified they only update the dashboard yaml. I was able to remove the table